application.py

```diff
@@ -703,6 +703,27 @@ def resource_by_template(primary_key_value=None):
 
         return full_rsp
 
+@application.route('/articles', methods=['GET'])
+def get_articles():
+    try:
+        #TODO change to request[password] and do jwt.decode()
+        password = "Yunho123"
+        user = UsersRDB.find_user(password)
+        results = UsersRDB.find_postinfo(user)
+
+        rsp_status = 504
+        full_rsp = Response(results, status=rsp_status, content_type="application/json")
+        full_rsp.headers["Access-Control-Allow-Origin"] = "*"
+
+        return full_rsp
+    except Exception as e:
+        rsp_txt = "Internal Error"
```

Comment:
your error message should never be "internal error"; even a "not found" could be more informative in this case